tools/sgcollect_info

```diff
@@ -738,7 +747,7 @@ def main():
 
     # If user asked to just upload, then upload and exit
     if options.just_upload_into is not None:
-        do_upload_and_exit(args[0], options.just_upload_into, options.upload_proxy)
+        do_upload_and_exit(args[0], options.just_upload_into, options.upload_proxy, keep_zip)
```

Comment:
I think here we could just avoid deleting it.

Reply:
https://github.com/couchbase/sync_gateway/pull/6119/commits/4bb03785f8f89dda3648cfe2c45fca796c80ff2f